website/docs/reference/node-selection/defer.md

```diff
@@ -33,6 +33,12 @@ $ dbt test --models [...] --defer --state path/to/artifacts
 ```
 
 </VersionBlock>
+<Changelog>
+
+- **v1.4**: Added the `--favor-state` option, which enables `--defer` to favor using the `--state` node even if the node exists in the current target. Using the `--favor-state` flag renders number 2 in the following criteria optional.
```

Comment:
```suggestion
- **v1.4**: Added the `--favor-state` option, which enables `--defer` to favor using the `--state` node even if the node exists in the current target. Using the `--favor-state` option renders the second criteria in the following optional.
```

Reply:
- sugg for consistency for translation purposes
- sugg for maybe better readability